hippod/api_object_post.py

```diff
@@ -290,6 +290,34 @@ def object_index_initial_add_major(app, sha_major, xobj):
     object_index_update(app, object_index)
 
 
+def check_anchor(app, xobj):
+    anchor = False
+    achievements = xobj['achievements']
+    for achievement in achievements:
+        if 'anchor' in achievement:
+            anchor = True
```

Comment:
no, not that way, if you spotted the anchor one time you can instantly return true, why iterating 1000 times again for no gain? change to return True, add to the end a return False, remove the local variable anchor and fine